components/ViewBills/ViewBills.js

```diff
@@ -103,6 +106,26 @@ const ViewBills = (props) => {
       <Row>
         {loading && <Spinner animation="border" className="mx-auto"/>}
       </Row>
+      <div className="d-flex justify-content-center mb-3">
+        <Button 
+        variant="primary" 
+        style={{marginRight: 15}}
+        onClick={previousPage}
+        disabled={!hasPreviousPage}
+        >
+          <FontAwesomeIcon icon={faAngleLeft} />
+        </Button>
+        <span>Page {currentPage}</span>
+        <Button 
+        variant="primary" 
+        style={{marginLeft: 15}}
+        disabled={false}
```

Comment:
Just need to remove this line to fix the linter check

Reply:
I feel very dumb for leaving that line in now that I see it lol